### PR TITLE
Issue #485: Slightly improved status report page (2 columns variant).

### PR DESCRIPTION
**Pull request by klonos**
Fixes https://github.com/backdrop/backdrop-issues/issues/485

Alternative to https://github.com/backdrop/backdrop/pull/1802

screen shot 2017-04-13 at 4 56 00 pm

### CODE REVIEW

File: core/modules/system/system.theme.inc [hunk 1]
```diff
@@ -294,26 +294,50 @@ function theme_status_report($variables) {
...
+          // ...and set the "merge" attribute to "up".
+          'merge' => 'up',
+        );
       }
     }
   }
 
-  $output .= '</table>';
-  return $output;
+  return theme_table($system_status_report_table);
```
Comment (quicksketch) [inline core/modules/system/system.theme.inc:340]: Just a note, this should be `theme('table', $system_status_report_table)` to make themeable.

File: core/modules/system/system.theme.inc [hunk 2]
```diff
@@ -294,26 +294,50 @@ function theme_status_report($variables) {
...
-  $output = '<table class="system-status-report">';
+
+  $system_status_report_table = array(
+    'header' => array(),
+    'rows' => array(),
+    'attributes' => array('class' => 'system-status-report'),
+    'caption' => '',
+    'colgroups' => array(),
+    'sticky' => FALSE,
+    'empty' => 'No status report. Something is definitely wrong.',
```
Comment (quicksketch) [inline core/modules/system/system.theme.inc:305]: We can probably skip this empty text.

File: core/includes/theme.inc [hunk 3]
```diff
@@ -2367,17 +2374,6 @@ function theme_item_list($variables) {
...
- */
-function theme_help($variables) {
-  // Wrap in a paragraph tag if one is not present.
-  if (strpos($variables['markup'], '<p') === FALSE) {
-    $variables['markup'] = '<p>' . $variables['markup'] . '</p>';
-  }
-  return '<div class="help">' . $variables['markup'] . '</div>';
-}
-
-/**
```
Comment (quicksketch) [inline core/includes/theme.inc:2380]: Oops, this looks like it may be a merge issue. We shouldn't be deleting `theme_help()`.

**Review by quicksketch — CHANGES_REQUESTED** [review]

**Comment by jenlampton** [comment]
closing in favor of https://github.com/backdrop/backdrop/pull/1836

**Pull request closed without merging**